src/Symfony/Bridge/Doctrine/Form/ChoiceList/DoctrineChoiceLoader.php

```diff
@@ -142,9 +115,9 @@ public function loadChoicesForValues(array $values, callable $value = null)
                 }
             }
 
-            return $objects;
+            return $objects ?? [];
```

Comment:
Rather than dealing with an undefined `$objects` variable, I would rather keep the empty array initialization.

Reply:
Thanks for the review. I've learned from @jpauli many years ago that this way will be compiled in a more optimized way and will avoid the "cost" of an initialization when it's not necessary.
I'm fine with changing it but I would rather encourage such writing whenever this is not subject to conflict with lower branches.

Reply:
Any thoughts? Is this a blocker? Maybe we could move forward on this one.

Reply:
I agree with @stof, we try to avoid using `??` with uninitialized variables usually.

Reply:
Alright, fixed.